cni/plugin.go

```diff
@@ -174,12 +174,14 @@ func (plugin *Plugin) InitializeKeyValueStore(config *common.PluginConfig) error
 		}
 	}
 
-	// Acquire store lock.
-	if err := plugin.Store.Lock(store.DefaultLockTimeout); err != nil {
-		log.Printf("[cni] Failed to lock store: %v.", err)
-		return err
+	// Acquiring store lock at this stage for optimization purpuses on Windows
```

Comment:
nit pic: spell check purposes and also add TODO, should remove this condition in future